analyze/analyses/dominators/mod.rs

```diff
@@ -0,0 +1,66 @@
+use std::collections::BTreeMap;
+
+use regex;
+
+use twiggy_ir as ir;
+use twiggy_opt as opt;
+use twiggy_traits as traits;
+
+use crate::analyses::utils;
+
+mod emit;
+
+struct DominatorTree {
+    tree: BTreeMap<ir::Id, Vec<ir::Id>>,
+    items: Vec<ir::Id>,
+    unreachable_items: Vec<ir::Id>,
```

Comment:
This field is new, this is where we store a list of unreachable items' Ids.

Reply:
Yeah so here is where I'd suggest we store the size and percent of unreachable items, since that is all we ever display and we wouldn't have to duplicate the logic of computing it in each of the `emit_*` methods